lib/dynflow/execution_plan.rb

```diff
@@ -547,6 +554,21 @@ def unlock_all_singleton_locks!
       end
     end
 
+    def toggle_telemetry_state(original, new)
+      return if original == new
+      @label = root_plan_step.action_class if @label.nil?
+      Dynflow::Telemetry.with_instance do |t|
+        t.set_gauge(:dynflow_active_execution_plans, '-1',
+                    telemetry_common_options.merge(:state => original)) unless original.nil?
+        t.set_gauge(:dynflow_active_execution_plans, '+1',
+                    telemetry_common_options.merge(:state => new)) unless new.nil?
+      end
+    end
```

Comment:
Do you ever send absolute number? I mean, relative is the correct approach but is it by default set to zero? I hope so

Reply:
No, just increment and decrement by 1. From what I've seen when implementing this the gauge starts at 0

Reply:
Great!